ios/engine/KMEI/KeymanEngine/Classes/fr.lproj/ResourceInfoView.strings

```diff
@@ -0,0 +1,3 @@
+
+/* Class = "UILabel"; text = "Scan this code to load this keyboard on another device"; ObjectID = "z2O-MT-IoV"; */
+"z2O-MT-IoV.text" = "Scan this code to load this keyboard on another device";
```

Comment:
and this, is it possible that our Crowdin upload tool has been missing the base localization file?  No one's localized it, despite covering most everything else.

Reply:
Crowdin has the French string
https://translate.keyman.com/translate/keyman/448/en-fr?filter=basic&value=3

Reply:
Found the issue and nixed it already.  The .yml file had an error.